content/components/only-manipulate-the-dom-via-the-renderer.md

```diff
@@ -0,0 +1,72 @@
+---
+title: only manipulate the DOM via the Renderer
+author: 
+  name: Billy Lando
+  url: https://github.com/billyjov
+---
+
+# Problem 
+
+According to the Angular documentation, relying on direct DOM access creates tight coupling between your application and rendering layers which will make it impossible to separate the two and deploy your application into a web worker. In addition, tight coupling restricts us from using **Ahead-of-Time** compilation which has obvious benefits, such as smaller bundle size or faster load times.
+
+Consequently, using jQuery , `document` object, or `ElementRef.nativeElement` is not recommended as it's not available on other platforms such as the server or web worker.
```

Comment:
Maybe clarify what 'platform server' means. I'm not sure everyone will know this refers to SSR.

Reply:
Good point!